pkg/uiserver/embedded_assets_rewriter.go

```diff
@@ -3,38 +3,41 @@
 package uiserver
 
 import (
+	"bytes"
+	"compress/gzip"
 	"html"
-	"os"
+	"io/ioutil"
 	"strings"
 )
 
-type modifiedFileInfo struct {
-	os.FileInfo
-	size int64
-}
-
-func (f modifiedFileInfo) Size() int64 {
-	return f.size
-}
-
-func (f modifiedFileInfo) Sys() interface{} {
-	return nil
-}
-
 func InitAssetFS(prefix string) {
```

Comment:
let's rename it :) @TennyZhuang  is not good at giving names.

How about `RewriteAssetsPublicPath(publicPath string)`

Reply:
done

Reply:
main package really don’t need to know what uiserver do with prefix, the InitAssetFs is more extensible, maybe we will use other implementation instead of rewrite.

I’m good at giving names, renaming here just “make reviewers happy”.